custom-steps/ffmpeg/pre-build.ps1

```diff
@@ -0,0 +1,12 @@
+. "$PSScriptRoot\..\..\util.ps1"
+
+$IsOnMacOS = Check-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "Not on Mac OS.  Exiting..."
```

Comment:
Maybe indicate why we're exiting? Or leave a comment saying why and then not Write-Host anything at all?

Reply:
Fixed: https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/1/commits/e19be2ec96316ea9170742f02b82dcb7bd4ca854